doc/dev/development.md

```diff
@@ -1,336 +1,12 @@
 # Development Guide
```

Comment:
it references local files like 
 - [Writing Mixer adapters](./adapters.md)
(hard to spot in the sea of red but it's there)
so I don't think we should do that